### DISCUSSION TITLE
[9 Settembre 2022] Pipeline

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/architettura-degli-elaboratori#82] [Pipeline (M-Z)]
![Esame](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/fc8c8618-77ee-47f8-b933-0205cee46a1b)
![Esame](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/804781a5-1ae5-435f-aa70-0608d6ec304a)
![Architettura Pipeline](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/176ea4e3-69cd-4fe8-ac0f-4fd7cb47e7df)
---
### Soluzione del Prof
Le soluzioni fornite dal professore potrebbero essere incomplete o poco chiare, quindi sono benvenute soluzioni che le completano, oppure soluzioni alternative e/o con spiegazioni più approfondite: [Soluzione.pdf](https://github.com/sapienzastudentsnetwork/archelab2223/files/11666384/AE.2021-22.-.Appello.3.-.2022-09-09.-.Esercizio.pipeline.-.Scheda.di.calcolo.pdf)

**Comment by Frisbon** [comment]
Ho conosciuto il tipo più fregno in palestra oggi mammamia che omo rega 

[Sono gay ne vado fiero](https://github.com/sapienzastudentsnetwork/archelab2223/files/11884000/Sono.gay.ne.vado.fiero.pdf)

**Reply by matypist** [reply]
Confermo la proposta di soluzione di @Frisbon, tranne per il punto 5, per cui confermo il risultato di @davidecimmarrusti (che coincide, tra l'altro, anche con il risultato del professore)

**Comment by davidecimmarrusti** [comment]
> Ho conosciuto il tipo più fregno in palestra oggi mammamia che omo rega
> 
> [Sono gay, ne vado fiero](https://github.com/sapienzastudentsnetwork/archelab2223/files/11884000/Sono.gay.ne.vado.fiero.pdf)

Segnalo un errore nel tuo punto 5.
Ho svolto l'esercizio e mi viene: 
IF: bolla, ID: lb $s0, ($a0), EX: j cycle, MEM: addi $a0,$a0,1, WB: bolla
